composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "shardj/zf1-future",
+    "name": "sandervanleeuwen/zf1-future",
```

Comment:
Just need to remove this change and I'll merge the PR

Reply:
Whoops. I changed the name in order to use my fork as an alternative for composer until these changes have been merged, but should probably have done so on a different branch. Reverted it now.

Reply:
this happened because you created pull request from your master branch.